model/client.go

```diff
@@ -1184,3 +1184,88 @@ func (c *Client) makeSecurityCall(resourceType, id, securityType, action string)
 	}
 
 }
+
+// MigrateClusterInstallation requests the migration of cluster installation(s) from the configured provisioning server.
+func (c *Client) MigrateClusterInstallation(request *MigrateClusterInstallationRequest) error {
+	resp, err := c.doPost(c.buildURL("/api/cluster_installations/migrate"), request)
+	if err != nil {
+		return nil
+	}
+	defer closeBody(resp)
+
+	switch resp.StatusCode {
+	case http.StatusOK:
+		return nil
+
+	default:
+		return errors.Errorf("failed with status code %d", resp.StatusCode)
+	}
+}
+
+// MigrateDNS requests the migration of cluster installation(s) from the configured provisioning server.
+func (c *Client) MigrateDNS(request *MigrateClusterInstallationRequest) error {
+	resp, err := c.doPost(c.buildURL("/api/cluster_installations/migrate/dns"), request)
+	if err != nil {
+		return nil
+	}
```

Comment:
We should not return `nil` if the error occurred. 
I see it in all new client methods so I assume it was copy & paste issue 🙂

Reply:
Yeah, it was following the existing pattern :) . Fixed.